controller/engine_controller.go

```diff
@@ -1237,6 +1255,14 @@ func handleRestoreError(log logrus.FieldLogger, engine *longhorn.Engine, rsMap m
 
 	for _, re := range taskErr.ReplicaErrors {
 		if status, exists := rsMap[re.Address]; exists {
+			if strings.Contains(re.Error(), restoreGetLockFailedMsg) {
+				// Register the name with a restore backoff entry
+				log.WithError(re).Debugf("Ignore failed locked restore error from replica %v", re.Address)
```

Comment:
nit: for logging, suggest below patterns instead of using the general verb, because it will confuse people if the action has been taken or not. 

- `verb+ing` before the action
- `verb+ed` after the action
- `failed to verb` failed the action